src/components/InMyFridge/RecipeList/RecipeModal/RecipeModal.js

```diff
@@ -0,0 +1,239 @@
+              <div 
+                className={styles.divImg}
+                style={{ backgroundImage:`url(${recipe.map(element => element.image)})`,
+                backgroundSize: 'cover',
+                backgroundRepeat: 'no-repeat',
+                backgroundPosition: 'center',
+                height: "300px",
+                width: "885px"
+                  }}
+              />
+
+              <div className={styles.RecipeTitle}>
+                <div>
+                  <h1 className={styles.title} id="transition-modal-title">{recipe.map(element => element.title)}</h1>
+                </div>
+                <div className={styles.RecipeIcon}>
+                  <div>
+                      <img src="../../../../Images/Icone/icon-cook-time@2x.png"alt="icone-cooking-min"/>
+                      {recipe.map((element, index) => element.cookingMinutes !== undefined
+                      ? <small key={index}>Cook {recipe.map(element => element.cookingMinutes)} min</small>
+                      : <small>no cooking time</small>
+                      )} 
+                  </div>
+                  <div>
+                    <img src="../../../../Images/Icone/icon-people-count@2x.png"alt="icone-cooking-min"/>
+                    <small>{recipe.map(element => element.servings)} people</small>
+                  </div>
+                  <div>
+
+
+                  <img src="../../../../Images/Icone/icon-prep-time@2x.png"alt="icone-cooking-min"/>
+                    {recipe.map((element, index) => element.preparationMinutes !== undefined
+                    ? <small key={index}> Prep {recipe.map(element => element.preparationMinutes)} min</small>
+                    : <small>no prep. time</small>
+                    )}
+                  </div>
+
+                </div>
+              </div>
+                <div className={styles.NutritionContainer}>
+                 {/* If its a complexe nutrition, set specific categories, if not set the other one */}
+                  {
+                    props.complexeNutrition
+                    ?  props.complexeNutrition.map((element, index) => {
+                      return <NutritionInfo
+                              key={index}
+                              icon={element.title}
+                              title={element.title}
+                              amount={element.amount}
+                              unit={element.unit}
+                      />
+                    })
+
+                    : diet.map((element, index) => {
+                      return <NutritionInfo
+                              key={index}
+                              icon={element.title}
+                              title={element.title}
+                              amount={element.amount}
+                              unit={element.unit}
+                      />
+                    })
+                  }                   
```

Comment:
Pour rendre le code plus clean : 

On crée une constante : 
` const isComplexeNutrition = props.complexeNutrition ? props.complexeNutrition : diet; `
````suggestion
                  {
                    isComplexeNutrition.map((element, index) => {
                      return <NutritionInfo
                              key={index}
                              icon={element.title}
                              title={element.title}
                              amount={element.amount}
                              unit={element.unit}
                      />
                    })
                  }         
```